mkdocs.yml

```diff
@@ -10,29 +10,31 @@ plugins:
 
 nav:
 - Home: 'index.md'
-- AppDev:
+- AI/ML:
```

Comment:
i would write this out and put the "AI/ML" in parens.. should help with SEO and newbies